lib/reducers/create-otp-reducer.js

```diff
@@ -135,10 +135,18 @@ export function getInitialState(userDefinedConfig) {
 
   const transitModeSettings = config?.modes?.transitModes.map((transitMode) => {
     const { mode, overrideMode } = transitMode
+    if (Array.isArray(mode) && !overrideMode) {
+      console.warn(
```

Comment:
When this warning fires the entire app does still crash but I guess the warning is there so that's good enough